versions.cfg

```diff
@@ -1,6 +1,7 @@
 [versions]
-six = 1.12.0
+plone.directives.dexterity = 1.0.2
```

Comment:
What do these version pinnings have to do with this commit which states that it will `Add category type.`? 

Please explain why these are needed in the commit message. You may want to move these changes to a separate commit where you can use its message body to elaborate the reason for the change.

Reply:
Checked the changes and determined that they are not necessary. Changes are removed now.